kore/src/main/scala/org/kframework/Strategy.scala

```diff
@@ -90,10 +90,6 @@ class Strategy(heatCool: Boolean) {
                     val strategy =
                       if (r.att.contains(Att.TAG)) {
                         makeRewrite(r.att.get(Att.TAG))
-                      } else if (heatCool && r.att.contains(Att.HEAT)) {
-                        makeRewrite("heat")
-                      } else if (heatCool && r.att.contains(Att.COOL)) {
-                        makeRewrite("cool")
```

Comment:
Here I've inlined the fact that `heatCool` is `false` by default, so removing the `--heat-cool-strategies` option makes these branches infeasible. Would like @dwightguth to confirm that this is intended behavior.